docusaurus/docs/iOS/customization/localization.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Translation and i10n
+---
+
+The SDK offers you in-house localization for all strings contained in UI components. For our internal localization, we provide `strings` file for each language that we translate. For advanced plural localization cases (typing indicator showing how many people are typing or how many responses are in thread), we use `stringsdict` with `NSStringPluralRuleType`. 
+
+## The Change of language on the device.
+
+Language for the StreamChat is changed together language of the device or simulator. See [how to change device language](https://support.apple.com/en-us/HT204031).
```

Comment:
Not sure how common / relevant this is, but what language does Stream Chat default to if the device is set to a language that we do not support?

Reply:
> Language for the StreamChat is changed together language of the device or simulator.

Change to: 

Stream Chat tries to use the same language as the device it is running on. If the device is set to a language that Stream Chat does not support, then Stream Chat will default to `${behavior the library takes when the device uses an unsupported language}`.

Reply:
Thank you for this, yes this makes more sense :)